schemas/dbt_project.json

```diff
@@ -139,6 +139,18 @@
                 }
             ]
         },
+        "label_configs": {
+            "title": "Label configs",
+            "type": "object",
+            "description": "Configurations specific to Big Query adapter used to add labels and tags to tables & views created by dbt.",
```

Comment:
Haven't commented this everywhere, but can you write the adapter name as BigQuery (no space) to match how Google calls it?

Reply:
Ah yes silly typo for me 😬 I've fixed that.